astrobee/config/tests/faults.config

```diff
@@ -1,182 +0,0 @@
--- Copyright (c) 2017, United States Government, as represented by the
```

Comment:
I changed the astrobee/config/tests/faults.config to be a simlink of the actual astrobee/config/faults.config, therefore no need to generate it. This was done to minimize error if someone manually changes this file for testing.